🚮 Fix 5509: remove iframe title attribute, set image alt to Instagram

lisaychuang opened the pull request:
🚮Fixes #5509
- remove iframe title attribute, as iframes are no longer exposed to AT 
- set image `alt` attribute to 'Instagram' instead of  `this.propagateAttributes(['alt'], image);`

Tagging @aghassemi @erwinmombay

aghassemi commented on extensions/amp-instagram/0.1/amp-instagram.js:
```diff
@@ -113,7 +113,7 @@ class AmpInstagram extends AMP.BaseElement {
     image.setAttribute('referrerpolicy', 'origin');
 
-    this.propagateAttributes(['alt'], image);
+    image.setAttribute('alt', 'Instagram');
```
Hi @lisaychuang 
Let's do `'Instagram: ' +	this.element.getAttribute('alt')` here so that page authors can also provide additional `alt`.

aghassemi commented:
@lisaychuang I also updated the description to `Fixes #5509` instead of `Fix issue 5509` since that particular format triggers GitHub's functionality to automatically close the issue when PR is merged!

lisaychuang commented:
@aghassemi thank you, I've updated my commit to reflect your suggested change!

aghassemi commented:
@lisaychuang LGTM, Thanks!

aghassemi commented:
@lisaychuang There are a few unit tests failure that need to be updated for the new behavior as well. 

```
DESCRIBE => amp-instagram
  DESCRIBE =>  
    IT => renders
      ✗ AssertionError: expected null to equal 'Instagram: Testing'
          at testIframe (home/travis/build/ampproject/amphtml/extensions/amp-instagram/0.1/test/test-amp-instagram.js:82:45)
          at http://localhost:9876/home/travis/build/ampproject/amphtml/extensions/amp-instagram/0.1/test/test-amp-instagram.js:94:7
●
    IT => renders captioned
      ✗ AssertionError: expected null to equal 'Instagram: Testing'
          at testIframeCaptioned (home/travis/build/ampproject/amphtml/extensions/amp-instagram/0.1/test/test-amp-instagram.js:89:45)
          at http://localhost:9876/home/travis/build/ampproject/amphtml/extensions/amp-instagram/0.1/test/test-amp-instagram.js:101:7
●●
    IT => builds a placeholder image without inserting iframe
      ✗ AssertionError: expected 'Instagram: Testing' to equal 'Testing'
          at testImage (home/travis/build/ampproject/amphtml/extensions/amp-instagram/0.1/test/test-amp-instagram.js:74:42)
          at http://localhost:9876/home/travis/build/ampproject/amphtml/extensions/amp-instagram/0.1/test/test-amp-instagram.js:119:7
          at http://localhost:9876/home/travis/build/ampproject/amphtml/extensions/amp-instagram/0.1/test/test-amp-instagram.js:63:9
●
    IT => removes iframe after unlayoutCallback
      ✗ AssertionError: expected null to equal 'Instagram: Testing'
          at testIframe (home/travis/build/ampproject/amphtml/extensions/amp-instagram/0.1/test/test-amp-instagram.js:82:45)
          at http://localhost:9876/home/travis/build/ampprojec
```

lisaychuang commented:
Oh, I do not have as much experience with testing so I'll try to find you during one of the breaks to work on this :) @aghassemi

aghassemi commented:
@lisaychuang There are three places in `https://github.com/ampproject/amphtml/blob/master/extensions/amp-instagram/0.1/test/test-amp-instagram.js` that are doing a `expect(iframe.getAttribute('title')).to.equal('Instagram: Testing');` 

Now that the value is on the placeholder image instead of the iframe, those lines need to change to do the assertion on the placeholder image itself.

lisaychuang commented:
Thanks for the tips @aghassemi , I've removed the `iframe` tests and added a new assertion for `testImage()` after referring to Chai JS doc. Hope this works!

aghassemi commented:
@lisaychuang Closing the PR since it is old, please reopen when you plan to work on it again. Thank you!
